microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -18,20 +18,20 @@ Review this article often.  The Microsoft Edge team updates this article as thin
 
 | Change | Stable Channel | Experimentation | Additional information |  
 |:--- |:--- |:--- |:--- |
-| Deprecate WebRTC's Plan B SDP semantics | [Chrome+1](#release-comments) \(Edge v94\)  |  | This change is happening in the Chromium project, on which Microsoft Edge is based. This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B. It is being replaced by the Unified Plan which is a spec-compliant and cross-browser compatible SDP format. For more information, navigate to [Chrome Platform Status entry][ChromestatusFeature5823036655665152] and [PSA: Timeline for Plan B SDP Deprecation and Removal - Please Migrate to Unified Plan][PSADeprecateWebRTCPlanB]. The Microsoft rollout schedule for deprecation is planned for one release after Chrome. Requesting a [WebRTC Plan B Reverse Origin Trial Token][ChromeDevelopersOrigintrialsWebRTCPlanBOriginTrial] allows sites to continue to use the deprecated API until Edge v96. |
-| Cookies default to `SameSite=Lax` and `SameSite=None-requires-Secure` | [Chrome+1](#release-comments) \(Edge v86\)  | Canary v82, Dev v82 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, navigate to the [Chrome Platform Status entry][ChromestatusFeature5088147346030592].  |  
-| Referrer Policy: Default to `strict-origin-when-cross-origin` | [Chrome+1](#release-comments) \(Edge v86\)  | Canary v79, Dev v79 | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, navigate to the [Chrome Platform Status entry][ChromestatusFeature6251880185331712].  |  
 | Disallow synchronous `XmlHttpRequest` in page dismissal | [Chrome+1](#release-comments) \(Edge v83\) |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  Matching Chrome, Microsoft Edge offers a Group Policy to turn off this change until Edge v88.  For more information, including the planned timeline by Google for this change, navigate to the [Chrome Platform Status entry][ChromestatusFeature4664843055398912].  |  
```

Comment:
change:
ms.date: 05/27/2021
to:
ms.date: 07/20/2021

Reply:
fixed